chapter_08_prettification.asciidoc

```diff
@@ -975,7 +1016,7 @@ INSTALLED_APPS = [
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "lists",
+    "lists.apps.ListsConfig",
```

Comment:
```suggestion
    "lists",
```

Reply:
Oh, I didn't notice this change. Got it.
https://github.com/hjwp/Book-TDD-Web-Dev-Python/commit/8c19394faf006933ff549ecb5f1bee943b0990a4#diff-7beedde00b2fc5c59611d91793c5cec65512ef6f7202caaa18980032a7ad4c1bR513